src/cli.nim

```diff
@@ -171,13 +177,13 @@ func genHelpText: string =
     optTrackDir: "Specify a track directory to use instead of the current directory",
     optVerbosity: &"The verbosity of output.\n" &
                   &"{padding}{allowedValues(Verbosity)} (default: normal)",
-    optSyncExercise: "Only operate on this exercise",
+    optFmtSyncExercise: "Only operate on this exercise",
+    optFmtSyncYes: "Auto-confirm every prompt for updating docs, filepaths, and metadata",
```

Comment:
Maybe do:

> optFmtSyncYes: "Auto-confirm every prompt for formatting docs, filepaths, and metadata",

Reply:
It's confusing, but this line is actually defining the description of the `--yes` option as used for `configlet sync`.

We're using `parseEnum` to get the option, so we can't have separate `optFmtYes` and `optSyncYes` both with the string value of `"yes"`. So right now, there's some mess to set a different description for `--yes` as an option for `fmt`, and I think we really do want different descriptions, because it's complicated that `--yes` doesn't auto-confirm a prompt for `--tests`

It's one of the things I was thinking of when I wrote in https://github.com/exercism/configlet/pull/468/commits/4e11cabd803e27d2956f01eec7320eb725b04e47:
> This could use some refactoring later. In particular, the generation
of the help message is becoming convoluted.

But "formatting docs, filepaths, and metadata" doesn't make sense, right? Because only `configlet sync` will operate on docs, and not `configlet fmt`, right? And when writing a `.meta/config.json` file with `configlet sync`, we try to avoid "formatting-only" changes, and only update when there was unsynced data.

With "updating" I was trying to imply that `--yes` is only meaningful as a `sync` option when `--update` is passed. Do you prefer e.g. "writing"?

Reply:
Thanks for explaining. Let's hold off on this for now.

Reply:
OK. Added a comment in https://github.com/exercism/configlet/pull/468/commits/94ff798d4967f67193d01187574f9b9730256374